platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectManager.java

```diff
@@ -63,6 +64,29 @@
     Project getProject(String workspace, String projectPath) throws ForbiddenException, ServerException, NotFoundException;
 
     /**
+     * Configure project after import. If project wasn't found in workspace it will be configured as blank.
+     *
+     * @param storage
+     *         storage from which project was imported
+     * @param workspaceId
+     *         workspace id
+     * @throws ServerException
+     *         if other error occurs
+     * @throws IOException
+     * @throws ForbiddenException
+     *         if user which perform operation doesn't have access to the requested project
+     * @throws NotFoundException
+     *         when project not found
+     * @throws ConflictException
+     *         if operation causes conflict
+     */
+    void configureProjectAfterImport(SourceStorageDto storage, String workspaceId) throws ServerException,
```

Comment:
Also does it really matter to configure project exactly after the import ?
This method may appear in sort of *Trigger* interface, but it is a bit strange to see method with name like this in the `ProjectManager` interface.